CONTRIBUTING.md

```diff
@@ -141,6 +141,12 @@ $ TEST_DEBUG=true make test
 
 You can combine `TEST_DEBUG` with `TEST_GREP` or `TEST_ONLY` to debug a subset of tests. If you plan to stay long in the debugger (which you'll likely do!), you may increase the test timeout by editing [test/mocha.opts](https://github.com/babel/babel/blob/master/test/mocha.opts).
 
+Use the `INCLUDE_STANDALONE` variable to run also `babel-standalone` tests:
```

Comment:
One option that I was thinking about is that we could alternatively have the tests actually run normally like they do now for all other packages, where rather than importing the Webpack bundle, they could run against `lib/index.js`, then the tests can all run as normal, and it's just the final bundling process that we skip locally. I don't think bootstrapping standalone is really that much of a time sink, but bundling is definitely slow and realistically all it's doing is testing Webpack features that don't need testing.